perseo-main/src/main/java/com/telefonica/iot/perseo/LogLevelServlet.java

```diff
@@ -95,11 +103,18 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
         logger.debug("getting log level");
         synchronized (mutex) {
-            String currentLevel = LogManager.getRootLogger().getLevel().toString();
-            response.setCharacterEncoding("UTF-8");
-            response.setContentType("application/json");
-            response.setStatus(HttpServletResponse.SC_OK);
-            response.getOutputStream().print("{\"level\":\""+ currentLevel +"\"}");
+        	 try {
```

Comment:
Wrong indentation

Reply:
Fixed in 61b8a4c9c7fe92f547917fd56b451a77f39db5f5